### DISCUSSION TITLE
sendAudio 仅支持什么文件的音频？？

### DISCUSSION THREAD

**Discussion by 347180440** [DreamVoid/MiraiMC#465] [问答 / Q&A]
![image](https://github.com/DreamVoid/MiraiMC/assets/107541071/4dcc3960-c91c-4b7e-886b-207ba83ab8ea)
ogg 还是没声音 mp3 也没声音
 String filePath2 = "C:\\Users\\Administrator\\Documents\\语音\\audio2.ogg";
                                        String outputFilePath = "C:\\Users\\Administrator\\Documents\\语音\\audio2.mp3";

                                        // 调用FFmpeg进行音频转码
                                        try {
                                            ProcessBuilder processBuilder = new ProcessBuilder("ffmpeg", "-i", filePath2, outputFilePath);
                                            Process process = processBuilder.start();
                                            process.waitFor();
                                        } catch (IOException | InterruptedException e) {
                                            e.printStackTrace();
                                        }

                                        // 发送转码后的MP3文件到群组
                                        File audioFile = new File(outputFilePath);
                                        MiraiBot.getBot(yourBotID).getGroup(targetGroupId).sendAudio(audioFile);

**Comment by DreamVoid** [comment]
MiraiMC的sendAudio方法为封装mirai的同名方法，仅作参数传递，具体的音频处理为mirai完成，建议查阅mirai的文档或寻求mirai的帮助。

**Reply by 347180440** [reply]
原来是电脑听不了 手机用户才能听
